assets/config.json

```diff
@@ -1,21 +1,22 @@
 {
-  "portal_user": "sw23",
-  "portal_passwd": "fake",
-  "portal_hostname": "localhost",
-  "portal_db_name": "aperture",
-  "portal_schema": "aperture",
+  "portal_user": "IlliaS",
+  "portal_passwd": "IlliaS",
+  "portal_hostname": "westbot.westus.cloudapp.azure.com",
+  "portal_db_name": "IlliaS_db",
+  "portal_schema": "test_aperture",
```

Comment:
uhm yeah, definitely should use a different schema config so it doesn't delete my aperture schema that took all night to write x)

Reply:
Also I mentioned this to Illia already but this config.json shouldn't be committed with database info on it, and also github will automatically run these tests as it is. We should consider whether we want github to run these e2e tests (I think yes), and if so, we should check in a different config.json with e2e test configs on it and keep it separate from the main config.json.